docs/END_USER_DOCUMENT.md

```diff
@@ -44,6 +44,7 @@ Mocca offers support for:
     1. CompletableFuture
     1. Pluggable asynchronous HTTP clients
     1. User provided executor services
+1. Request Parameter Validation
 
```

Comment:
Please add the new section at the top of the document, under the Summary.

Reply:
of course. sorry about that.